src/routes/+page.svelte

```diff
@@ -158,4 +158,9 @@
 			url("../lib/assets/images/background-pattern.jpg");
 		background-size: 70%;
 	}
+	@media only screen and (max-width: 800px) {
```

Comment:
Could we refactor this to use `min-width`?

It is considered [best practice](https://stackoverflow.com/questions/16647380/max-width-vs-min-width#:~:text=Typically%2C%20with%20min%2Dwidth%20patterns,then%20target%20progressively%20larger%20screens.) and allows for mobile-first styling by default.